contracts/common/Registry.sol

```diff
@@ -126,6 +127,10 @@ contract Registry is Governable {
         return contractMap[STAKE_MANAGER];
     }
 
+    function getSlashingManagerAddress() public view returns (address) {
```

Comment:
From i've seen in the code - some getters follow solidity way - without using "get" and some code uses "get". I don't have preferences here, but it would be awesome if the code follows same rules.

Reply:
yes, I'll refactor it in a separate PR.